docs(wiki): fix typos, improve formatting/readability (#129)
BlueMap-Minecraft/BlueMapWiki: community/Chat.md

Requirement keywords as per [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119)

- You **MUST** have a Spigot based server. Paper **SHOULD** be used. The addon doesn't support other platforms.
- You **MUST** have root access and be able to install and configure additional software such as nginx.

> - You **MUST NO**T run a server network with Bungeecord, Velocity or similar. The addon only supports one global chat.

Reviewer: ```suggestion
- You **MUST NOT** run a server network with Bungeecord, Velocity or similar. The addon only supports one global chat.
```

Author: Whoops! Good catch!
